libs/Microsoft.MixedReality.WebRTC/AudioTransceiver.cs

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Might seem a dejavu, but... can we have a single class for Transceiver? Methods could take a MediaTrack and then cast it whenever necessary.

Reply:
Yes I hesitated but I am more inclined to keep the type safety here. Would a generic class be better?

Reply:
Yes, I don't think this gives us much safety - the only thing that it prevents is passing an audio track to a video transceiver, which seems quite easy to avoid anyway. I am much more worried about bugs caused by duplicate code getting out of sync.

Reply:
Fair enough, will try merging and see how it looks.

Reply:
Looks good enough. Fixed.